apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.html.ts

```diff
@@ -0,0 +1,316 @@
+      The <code>instance</code> column indicates which assessment instance to modify, and should be
+      <code>1</code> if there is only a single instance per student.
+    </p>
+    <p>
+      Alternatively, the total assessment points can be changed with a CSV containing a
+      <code>points</code> column, like:
+    </p>
+    <pre class="ml-4">
+uid,instance,points
+student1@example.com,1,120
+student2@example.com,1,130.27</pre
+    >
+    <p>For assessments using group work, use the <code>group_name</code> column instead:</p>
+    <pre class="ml-4">
+group_name,instance,score_perc
+myhappygroup,1,95
+greatgroup,1,85</pre
+    >
+  `;
+}
+
+function UploadInstanceQuestionScoresModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'upload-instance-question-scores-form',
+    title: 'Upload new question scores',
+    formEncType: 'multipart/form-data',
+    body: html`
+      ${CsvHelpInstanceQuestionScores()}
+      <div class="form-group">
+        <div class="custom-file">
+          <input
+            type="file"
+            name="file"
+            class="custom-file-input"
+            id="uploadInstanceQuestionScoresFileInput"
+          />
+          <label class="custom-file-label" for="uploadInstanceQuestionScoresFileInput">
+            Choose CSV file
+          </label>
+        </div>
+      </div>
+      <div class="d-flex justify-content-end">
+        <div class="form-group mb-0">
+          <input type="hidden" name="__action" value="upload_instance_question_scores" />
+          <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+          <button type="submit" class="btn btn-primary">Upload</button>
+        </div>
+      </div>
+    `,
+  });
+}
+
+function UploadAssessmentInstanceScoresModal({ csrfToken }: { csrfToken: string }) {
+  return Modal({
+    id: 'upload-assessment-instance-scores-form',
+    title: 'Upload new total scores',
+    formEncType: 'multipart/form-data',
+    body: html`
+      ${CsvHelpAssessmentInstanceScores()}
+      <div class="form-group">
+        <div class="custom-file">
+          <input
+            type="file"
+            name="file"
+            class="custom-file-input"
+            id="uploadAssessmentInstanceScoresFileInput"
+          />
+          <label class="custom-file-label" for="uploadAssessmentInstanceScoresFileInput"
+            >Choose CSV file</label
+          >
+        </div>
+      </div>
+      <div class="d-flex justify-content-end">
+        <div class="form-group mb-0">
+          <input type="hidden" name="__action" value="upload_assessment_instance_scores" />
+          <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+          <button type="submit" class="btn btn-primary">Upload</button>
+        </div>
```

Comment:
Ditto

Reply:
Moved this to the footer in [9e9167c](https://github.com/PrairieLearn/PrairieLearn/pull/9777/commits/9e9167c590c1e310b067653a78e2c2d3df5f5246).